pkg/envoy/lds/connection_manager.go

```diff
@@ -47,6 +51,16 @@ func getHTTPConnectionManager(routeName string, cfg configurator.Configurator, h
 		AccessLog: envoy.GetAccessLog(),
 	}
 
+	incomingExtAuthCfg := cfg.GetInboundExternalAuthConfig()
```

Comment:
Make the call to `GetInboundExternalAuthConfig()` only if the direction is inbound (minor perf optimization).

Reply:
the `enable` is in the config :), can't skip calling this guy